Add device metrics docs (#1930)
balena-io/docs: pages/reference/supervisor/bandwidth-reduction.md

# Reduce bandwidth usage

> In order to provide users with a wealth of information about their devices, the device's {{ $names.company.lower }} Supervisor regularly keeps the API informed about device state. Upon receiving target device state that differs from current device state, the Supervisor applies changes, such as downloading new application updates or setting environment variables. While a quick reflection of actions is great during development, it comes at a cost of increased data usage.

Reviewer: ```suggestion
In order to provide users with essential information about their devices, the {{ $names.company.lower }} Supervisor on the device regularly keeps the API informed about the device state. Upon receiving a target device state that differs from the current device state, the Supervisor applies changes, such as downloading new application updates or setting environment variables. While a quick response to actions is great during development, it also leads to increased data usage.
```

Reviewer 2: I like Vipul's suggested edit

Author: Changed with the exception of 
- `regularly keeps the API informed about the device state` --> `regularly keeps the API informed about the device's state`. 
- `a target device state that differs from the current device state` --> `a target device state which differs from the current device state`

| Variable                               | Allowed Value      |   Action                                             | Default | Available with Supervisor Version |
|----------------------------------------|--------------------|------------------------------------------------------|---------|-----------------------------------|
| `BALENA_SUPERVISOR_VPN_CONTROL`        | true/false         |  Enable / Disable VPN                                |  true   |  v1.1.0                           |
| `BALENA_SUPERVISOR_CONNECTIVITY_CHECK` | true/false         |  Enable / Disable VPN connectivity check             |  true   |  v1.3.0                           |
| `BALENA_SUPERVISOR_POLL_INTERVAL`      | 600000 to 86400000 |  API Poll interval in milliseconds                   |  900000 |  v1.3.0                           |
| `BALENA_SUPERVISOR_LOG_CONTROL`        | true/false         |  Enable / Disable logs from being sent to API        |  true   |  v1.3.0                           |
| `BALENA_SUPERVISOR_HARDWARE_METRICS`   | true/false         |  Enable / Disable device hardware metrics reporting  |  true   |  v12.8.0                          |

> Side Effects / Warning

Reviewer: ```suggestion
Side Effects / Warnings
```

`BALENA_SUPERVISOR_LOG_CONTROL`: Any logs written by the user container or the device Agent are not sent to the dashboard when this variable is set to False.

> `BALENA_SUPERVISOR_HARDWARE_METRICS`: [Device metrics][device-metrics] such as CPU temperature and memory usage won't be reported to the API, and won't be displayed in the device's dashboard summary page.

Reviewer: ```suggestion
`BALENA_SUPERVISOR_HARDWARE_METRICS`: [Device metrics][device-metrics] such as CPU temperature and memory usage won't be reported to the API, and won't be displayed on the summary page of the device dashboard.
```

|-------------------------------------------------------------------------|---------------------------------------------------------|--------------------------------------|
| API poll (Once per 15min)                                               | 6650 Bytes per request                                  | BALENA_SUPERVISOR_POLL_INTERVAL      |
| {{ $names.company.lower }} Supervisor update poll (Once every 24 hours) | 6693 Bytes per request                                  | Not configurable via variables       |
| VPN enabled                                                             | 43 Bytes / second                                       | BALENA_SUPERVISOR_VPN_CONTROL        |
| TCP check cost (When VPN disabled)                                      | 47.36 Bytes / second                                    | BALENA_SUPERVISOR_CONNECTIVITY_CHECK |
| Hardware metrics reporting                                              | ~66 Bytes every 10 seconds when there have been changes | BALENA_SUPERVISOR_HARDWARE_METRICS   |

Example minimum bandwidth settings
----------------------------------

> The following settings result in data usage of approximately 1.3MB per month:

Reviewer: ```suggestion
The following settings lead to data usage of approximately 1.3MB per month:
```

* Disable VPN
* Disable CONNECTIVITY CHECK
* Change POLL interval to 24 hours

> * Disable LOGS

Reviewer: Is there a reason why these 3 are capital, if they do symbolize a config var then please put them under backticks
